packages/webdriverio/tests/commands/element/moveTo.test.ts

```diff
@@ -9,43 +9,6 @@ vi.mock('got')
 vi.mock('@wdio/logger', () => import(path.join(process.cwd(), '__mocks__', '@wdio/logger')))
 
 describe('moveTo', () => {
-    it('should do a moveTo without params', async () => {
```

Comment:
Why do we remove these tests?

Reply:
Generally it's my bad, I've thought scroll can work only with e2e here, but the assumption was wrong, it can be run with mock like elem.elementId = { scrollIntoView: vi.fn() }. I've restored the test cases